packages/material-ui/src/Chip/Chip.d.ts

```diff
@@ -29,7 +30,9 @@ export type ChipClassKey =
   | 'label'
   | 'deleteIcon'
   | 'deleteIconPrimary'
-  | 'deleteIconSecondary';
+  | 'deleteIconSecondary'
```

Comment:
The PR adds 5 new classes. There is only two here.

Reply:
I just noticed that for deleteIconPrimary and deleteIconSecondary the .d.ts file doesn't match the implementation (deleteIconColorPrimary, deleteIconColorSecondary).
what should I do about it?